src/lib/components/BrandingFooter.tsx

```diff
@@ -0,0 +1,40 @@
+import { Trans } from '@lingui/macro'
+import Row from 'lib/components/Row'
+import { Logo } from 'lib/icons'
+import styled from 'lib/theme'
+
+import ExternalLink from './ExternalLink'
+
+const UniswapA = styled(ExternalLink)`
+  color: ${({ theme }) => theme.secondary};
+  cursor: pointer;
+  display: flex;
```

Comment:
Display and justify-content should be enabled through props in the Row, not specified explicitly.

Reply:
fwiw I really dislike this pattern of componentizing css rules.

Reply:
Much prefer interpolating css into the `styled` definition.